### PR TITLE
Remove starting csv

### PR DESCRIPTION
**Pull request by gparvin**
Remove the starting CSV entry when installing operators with a policy.  Update the channel when there's a better 
value too.

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -111,7 +111,6 @@ Policy  | Description | Prerequisites
 [Policy to install `cert-manager`](./CM-Configuration-Management/policy-cert-manager-operator.yaml) | Use this policy to deploy the community operator for `cert-manager` which installs `cert-manager` on OpenShift clusters. | For more information on `cert-manager` visit [Cloud native certificate management](https://cert-manager.io/)
 [Policy to label a Managed-Cluster](./CM-Configuration-Management/policy-label-cluster.yaml) | Use this policy to label a Managed-Cluster | This policy needs to be applied on the Managing-Cluster, adjust the labels to your needs
 [Policy to set a Config-Map with properties for different environments](./CM-Configuration-Management/policy-engineering-configmap.yaml) | Use this policy to configure a policy for different environments | adjust this example for your needs
-[Policy to install Local Storage Operator](./CM-Configuration-Management/policy-odf.yaml) | Use this policy to install and configure the Local Storage Operator | adjust the LocalVolumeSet and StorageClass for your needs
```
Comment (ycao56) [inline community/README.md]: Why is this removed?

Comment (gparvin) [reply community/README.md]: The OpenShift Container Operator policy `policy-ocs-setup.yaml` is the working storage policy.  The `odf` policy doesn't work and appears to overlap with OCS anyway.

Comment (ycao56) [reply community/README.md]: OK. @ch-stark contributed this policy. Can you confirm with him and see if he is ok with removing this policy?

Comment (gparvin) [reply community/README.md]: ok, sounds good, will check with him.

Comment (gparvin) [reply community/README.md]: @ycao56 @ch-stark I "undid" my removal of odf.  If that policy is no longer needed I can remove it later.

Comment (ycao56) [reply community/README.md]: cool

**Review by ycao56 — APPROVED** [review]
